[DOCS-7162] Add column width section (#966)
Alfresco/docs-alfresco: process-automation/latest/using/index.md

![Select columns]({% link process-automation/images/select-columns.png %})

### Column width

> You can adjust the width of the columns of the **Process Management** section. To do this select the edge of the columns and adjust them to the desired width. Once you have changed one column, all the columns with the same name are also adjusted.

Reviewer: You can add that the column size is saved in preferences. If user will log out and in, it should stay the same.

Author: Done.

> You can adjust the width of the columns of the **Process Management** section. To do this select the edge of the columns and adjust them to the desired width. Once you have changed one column, all the columns with the same name are also adjusted. If you log out and then log back in again the new column widths will be preserved.

Reviewer 2: Rewrite using present tense where possible, for example:
```suggestion
You can adjust the width of the columns of the **Process Management** section. To do this select the edge of the columns and adjust them to the desired width. Once you change one column, all the columns with the same name are also adjusted. If you log out and then log back in again, the new column widths is preserved.
```

This feature is enabled by default for the **Process Management** section. If you want to adjust the column widths in the same way for the **Personal Files** or **File Libraries** sections of the Digital Workspace, you must turn it on by editing the `libs/content-ee/process-services-cloud-extension/assets/process-services-cloud.extension.json` file and adding the following:

> ```JSON

Reviewer 2: Prefer lowercase markers:
```suggestion
```json
```
